File: Angular y Typescript/AppTestAvanced/src/app/persona.ts
export class Persona {

    nombre:string;
    apellidos:string;
    edad:number;
    informatico: boolean;

    constructor(nombre:string, apellidos:string, edad:number, informatico?: boolean) {

        this.nombre = nombre;
        this.apellidos = apellidos;
        this.edad = edad;
        this.informatico = true;
    
    }

    


}

File: Java+Ioc/eclipse/GestampTest/src/com/Gestamp/Ejercicio4/Persona.java
package com.Gestamp.Ejercicio4;

public class Persona {
	
	private int edad;
	private String nombre;
	
	//Relacion entre persona y telefono
	
	private Telefono telefono;
	
	public Persona(String nombre, int edad) {
		super();
		this.edad = edad;
		this.nombre = nombre;
	}
	
	public int getEdad() {
		return edad;
	}
	public void setEdad(int edad) {
		this.edad = edad;
	}
	public String getNombre() {
		return nombre;
	}
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	public Telefono getTelefono() {
		return telefono;
	}
	public void setTelefono(Telefono telefono) {
		this.telefono = telefono;
	}
	
	public void muestraDatoPersona() {
		
		System.out.println("" + getNombre());
		System.out.println();
		
	}
	
}
